webpack.config.js

```diff
@@ -43,6 +43,7 @@ const shouldMinify =
 	process.env.MINIFY_JS === 'true' ||
 	( process.env.MINIFY_JS !== 'false' && bundleEnv === 'production' && calypsoEnv !== 'desktop' );
 const shouldEmitStats = process.env.EMIT_STATS && process.env.EMIT_STATS !== 'false';
+const shouldShowProgress = process.env.PROGRESS && process.env.PROGRESS !== 'false';
```

Comment:
If you no longer enable progress reporting on mere `EMIT_STATS=true`, then please add `PROGRESS=true` to the `preanalyze-bundles` npm task. So that the `icfy-stats` CircleCI job continues to log the progress, too.

Reply:
done